### DISCUSSION TITLE
Heatmap in candlestick

### DISCUSSION THREAD

**Discussion by jxg-source** [tradex-app/TradeX-chart#21] [Ideas]
Is it possible to add heatmap in candlestick, similar TradingLite?

**Comment by tradex-app** [comment]
At the moment, there is no heatmap feature.
However one could be built with the overlay API.
I took a look at https://www.tradinglite.com/
I guess you are referring to the Liquidity Heatmap.

**Reply by jxg-source** [reply]
Yes, I will forward it to you.
![image](https://github.com/tradex-app/TradeX-chart/assets/33990923/39516030-db32-4ad5-b5c3-d97e948c343f)

**Reply by tradex-app** [reply]
Cheers! Where do you get your orderbook historical data from?

**Reply by jxg-source** [reply]
It would be the orderbook on Binance: https://www.binance.com/en/landing/data
